### PR TITLE
Telegram mini-app deployment

### PR DESCRIPTION
**Pull request by antonyshchenko**

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -58,9 +62,20 @@ jobs:
...
       environment: test
 
+  deploy_telegram_mini_app:
+    name: Deploy Telegram Mini App
+    if: ${{ github.event_name == 'push' }}
+    needs: [ build_push_backend, build_telegram_mini_app ]
+    permissions:
+      id-token: write # needed for aws credentials
+      contents: read
+    uses: ./.github/workflows/deploy-web-ui.yml
```
Comment (imykolenko) [inline .github/workflows/CI.yml:72]: It looks like should be `deploy-telegram-mini-app`

```suggestion
    uses: ./.github/workflows/deploy-telegram-mini-app.yml
```

File: terraform/demo/main.tf [hunk 2]
```diff
@@ -230,6 +230,17 @@ module "web" {
   certificate_arn = data.aws_acm_certificate.chainring_us_east_1.arn
 }
 
+module "telegram_mini_app" {
+  source      = "../modules/telegram_mini_app"
+  name_prefix = local.name_prefix
+  zone        = data.terraform_remote_state.shared.outputs.zone
+  providers = {
+    aws.us_east_1 = aws.us_east_1
+  }
```
Comment (imykolenko) [inline terraform/demo/main.tf:239]: in `telegram_mini_app` there is no variable with name `providers`. What function it has?

File: .github/workflows/CI.yml (hunk 1 above)
Comment (antonyshchenko) [reply .github/workflows/CI.yml:72]: Yup, thanks

File: terraform/demo/main.tf (hunk 2 above)
Comment (antonyshchenko) [reply terraform/demo/main.tf:239]: I'm not sure, but if I remove it Terraform would complain. I guess it has something to do with `required_providers` in `modules/telegram_mini_app/versions.tf`

**Pull request merged**